src/cascadia/TerminalSettingsModel/IInheritable.h

```diff
@@ -49,7 +52,13 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             _parents.insert(pos, parent);
         }
 
-        std::vector<com_ptr<T>> ExportParents()
+        // Method Description:
+        // - Exports list of parents as an immutable view.

```

Comment:
not a view, not immutable. contractually "not going to change my copy", but you can still wreak havoc with the parents inside the list